client/overview/index.js

```diff
@@ -131,6 +132,10 @@ const OverviewPage = () => {
 
 			<TestModeNotice topic={ topics.overview } />
 
+			<ErrorBoundary>
```

Comment:
To keep the same pattern, what do you think about adding the `wcpaySettings.isFraudProtectionSettingsEnabled` flag check here?

Reply:
I've moved this check here in [3198f43](https://github.com/Automattic/woocommerce-payments/pull/5611/commits/3198f439f55fd25e710d2fb3007d13f8843f3203).